frontend/src/app/projects/project-detail/project-metadata/project-metadata.component.html

```diff
@@ -6,21 +6,45 @@
 <div class="wrapper">
   <h2 id="title">Metadata</h2>
   <mat-card>
-    <form>
-      <mat-form-field class="example-full-width" appearance="fill">
-        <mat-label>Description</mat-label>
-        <textarea
-          [formControl]="updateDescriptionForm"
-          matInput
-          rows="3"
-          style="resize: none"
-          [placeholder]="
-            'This is the description of the project ' + project!.slug
-          "
-        ></textarea>
-      </mat-form-field>
+    <form [formGroup]="form">
+      <fieldset>
+        <mat-form-field appearance="fill">
+          <mat-label>Name</mat-label>
+          <input matInput formControlName="name" />
+          <mat-error *ngIf="form.controls.name.errors?.required"
+            >The project name is required.</mat-error
+          >
+          <mat-error *ngIf="form.controls.name.errors?.uniqueSlug"
+            >A project with a similar name already exists.</mat-error
+          >
+          <mat-hint *ngIf="project.slug !== newSlug && form.controls.name.valid"
+            >If you proceed, the project slug will be changed to
+            <i>{{ newSlug }}</i> - Please make sure to update all
+            references.</mat-hint
+          >
+        </mat-form-field>
+      </fieldset>
+      <fieldset>
+        <mat-form-field appearance="fill">
+          <mat-label>Description</mat-label>
+          <textarea
+            formControlName="description"
+            matInput
+            rows="3"
+            style="resize: none"
```

Comment:
No harm done if we support resizing.

Reply:
Kind of, the tiles in the overview are not expandable, so too long descriptions will break them - but this is something we should fix.